lms/djangoapps/certificates/api.py

```diff
@@ -469,6 +469,16 @@ def get_active_web_certificate(course, is_preview_mode=None):
     configurations = certificates.get('certificates', [])
     for config in configurations:
         if config.get('is_active') or is_preview_mode:
+            # [UCSD_CUSTOM] remove '/' from signatory image path
+            # "Preview certificate" error fix
+            signatories = config.get('signatories')
+            for signatory in signatories:
+                signatory.update({
+                    'signature_image_path': signatory['signature_image_path'][1:]
```

Comment:
I think we should first verify that `signature_image_path` is starting with a `/` and only then truncate it. But if this is for the hotfix purposes only, then can be ignored for now I guess.

Reply:
I agree we should care about an edge case here, otherwise, it can raise an exception.